apps/api/docs/development-setup.md

```diff
@@ -40,22 +40,22 @@ Make sure Redis and MariaDB server are up and running.
    ```env
    # Server
    SERVER_PORT=3000
-   SERVER_API_KEY=osmo-notify-test-key
+   SERVER_API_KEY=osmoX-test-key
```

Comment:
here also make it `OsmoX` as this is more of a reference to OsmoX not value like `osmox-mariadb`.

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b